### ISSUE TITLE
Example app from How to GraphQL is outdated

### ISSUE THREAD

**Issue by bgentry** [bgentry/ember-apollo-client#122]
Hey @DevanB, just realized that the example app linked near the top of the readme is no longer up-to-date on best practices for this addon. Specifically, it’s using a really old version of the addon and hasn’t yet been upgraded to Apollo Client v2.

Do you or anybody else have the bandwidth to update the app & tutorial? Otherwise I may need to remove the link so it doesn’t lead new users astray.

**Comment by DevanB** [comment]
Yep! All of the How to GraphQL authors are working on updating all of the frontend tutorials to use Prisma and the latest versions of the Apollo addons.

I’m slated to update the tutorial in about 2-3 weeks. Feel free to remove temporarily if you’d like. :)

**Comment by datajohnny** [comment]
have the tutorials been updated?

**Comment by marsh73** [comment]
+1
